src/python/grpcio/grpc/_channel.py

```diff
@@ -142,7 +142,7 @@ def _handle_event(event, state, response_deserializer):
         response = _common.deserialize(
             serialized_response, response_deserializer)
         if response is None:
-          details = b'Exception deserializing response!'
+          details = 'Exception deserializing response!'
```

Comment:
Apologies for the post-integrate comment, but I don't think I understand the change on this line. `details` strings should always be bytes, right? Never native strings? Why drop the `b` prefix from this line?